deploy/charts/openebs-monitoring/rules/volumes/volume-node-rules.yaml

```diff
@@ -0,0 +1,47 @@
+groups:
+  - name: volume-node
+    rules:
+      # Alert for pv's fileystem getting into read-only state while re-mounting on a particular node.
+      - alert: VolumeNodeFileSystemIsReadOnly
+        annotations:
+          summary: 'Persistent Volume on node ''{{ $labels.node }}'' for persistent volume claim ''{{ $labels.persistentvolumeclaim }}'' is having read-only filesystem while re-mounting'
```

Comment:
Check the below message:
Volume mount failed for persistent volume claim {claim name} on node {node name} due to read-only file-system

Reply:
Looks good.

Reply:
I think summary. Do we have more data to show in description? Like the reason for the condition...

Reply:
Reason is itself mentioned as the alert name. The metrics labels which we get are:
{endpoint="https-metrics", instance="10.41.10.72:10250", job="kubelet", metrics_path="/metrics", namespace="default", node="node1-virtual-machine", persistentvolumeclaim="csi-lvmpv", reason="FilesystemIsReadOnly", service="openebs-monitor-kube-prome-kubelet"}
--